### PR TITLE
Add Swift service and nodeset sample

### PR DESCRIPTION
**Pull request by cschwede**
Depends-On: https://github.com/openstack-k8s-operators/edpm-ansible/pull/556

### CODE REVIEW

**Comment by cschwede** [comment]
recheck

File: config/services/dataplane_v1beta1_openstackdataplaneservice_swift.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: swift
+spec:
+  label: swift
+#  openStackAnsibleEERunnerImage: quay.io/cschwede/openstack-ansibleee-runner:test
```
Comment (fao89) [inline config/services/dataplane_v1beta1_openstackdataplaneservice_swift.yaml:7]: I think we can drop this

Comment (cschwede) [reply config/services/dataplane_v1beta1_openstackdataplaneservice_swift.yaml]: Oh yes, sorry for that - jus fixed and pushed an updated commit.

**Review by fao89 — APPROVED** [review]

**Comment by cschwede** [comment]
/test dataplane-operator-build-deploy-kuttl

the server is currently unable to handle the request (get users.user.openshift.io ~)

**Comment by cschwede** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by cschwede** [comment]
recheck

**Comment by cschwede** [comment]
recheck

Connection reset by 38.102.83.162 port 22\r\nrsync: connection unexpectedly closed (0 bytes received so far)

**Comment by fao89** [comment]
recheck

**Pull request merged**